### ISSUE TITLE
Fusion CLI assumes all node modules have no side effects

### ISSUE THREAD

**Issue by vicapow** [fusionjs/fusion-cli#630]
As of version 1.12.1, fusion-cli assumes all node_modules have no side effects, which is unsafe. I'll be providing more detail here but creating the ticket first to track.

**Comment by rtsao** [comment]
This should only be the case if `assumeNoImportSideEffects: true` is set. Is this flag enabled?

**Comment by vicapow** [comment]
yes, this flag is and was enabled across updates. this change in behavior may have been to fix a previous but but it introduced a breaking change in doing so.
